### PR TITLE
Template fixes and v1.4.0a2 release

### PR DESCRIPTION
**Pull request by kprzerwa**

### CODE REVIEW

File: invenio_accounts/templates/semantic-ui/invenio_accounts/login_user.html [hunk 1]
```diff
@@ -10,45 +10,48 @@
...
+      {%- block form_header %}
+        <h3 class="ui header">{{ _('Log in to account') }}</h3>
+      {%- endblock form_header %}
+
+      {%- block form_outer %}
+        {%- with form = login_user_form %}
+          <form action="{{ url_for_security('login') }}" method="POST"
+                name="login_user_form">
+            {{ form.hidden_tag() }}
+            {{ form_errors(form) }}
```
Comment (ntarocco) [inline invenio_accounts/templates/semantic-ui/invenio_accounts/login_user.html:25]: Could you please check if errors are rendered correctly?

File: invenio_accounts/templates/semantic-ui/invenio_accounts/login_user.html [hunk 2]
```diff
@@ -10,45 +10,48 @@
...
+            {{ render_field(form.email, icon="user icon", autofocus=True, errormsg=False) }}
+            {{ render_field(form.password, icon="lock icon", errormsg=False) }}
+            <button type="submit" class="ui fluid large submit primary button">
+              <i class="ui sign-in icon"></i>{{ _('Log In') }}
```
Comment (ntarocco) [inline invenio_accounts/templates/semantic-ui/invenio_accounts/login_user.html:29]: super minor: WDYT?
I know it was like this, but I was reading for ILS that it is preferable the Sign In form... plus I see Sign up below?
```suggestion
              <i class="ui sign-in icon"></i>{{ _('Sign In') }}
```
And maybe to change it also in the header to keep it consistent?

File: invenio_accounts/templates/semantic-ui/invenio_accounts/register_user.html [hunk 3]
```diff
@@ -11,42 +11,46 @@
...
+        <h3 class="ui header">
+          {% trans sitename=config.ACCOUNTS_SITENAME %}Sign up for a
+            {{ sitename }} account!{% endtrans %}
+        </h3>
+      {%- endblock form_header %}
+
+      {%- with form = register_user_form %}
+        <form action="{{ url_for_security('register') }}" method="POST"
+              name="register_user_form">
+          {{ form_errors(form) }}
```
Comment (ntarocco) [inline invenio_accounts/templates/semantic-ui/invenio_accounts/register_user.html:28]: same here for form errors

File: invenio_accounts/templates/semantic-ui/invenio_accounts/register_user.html [hunk 4]
```diff
@@ -11,42 +11,46 @@
...
+            <i class="ui edit icon"></i>{{ _('Sign Up') }}
+          </button>
+        </form>
+      {%- endwith %}
+      <div class="divider hidden"></div>
+    </div>
+
+    <div class="ui secondary segment padded">
+      {{ _('Already have an account?') }}
+      <a href="{{ url_for_security('login') }}">{{ _('Log In') }}</a>
```
Comment (ntarocco) [inline invenio_accounts/templates/semantic-ui/invenio_accounts/register_user.html:52]: in case you go for Sign in, here too

**Comment by kprzerwa** [comment]
to fix form errors display, issue created: https://github.com/inveniosoftware/invenio-accounts/issues/325

**Pull request merged**